interface/super/edit_layout.php

```diff
@@ -114,7 +116,7 @@ function genGroupSelector($name, $layout_id, $default = '')
         array($layout_id)
     );
     $s  = "<select class='form-control' name='" . xla($name) . "'>";
-    $s .= "<option value=''>" . xlt('None') . "</option>";
+    $s .= "<option value=''>" . xlt('None{{Group}}') . "</option>";
```

Comment:
None of these labels should go into the patch or they will break translations for those terms. All that stuff will need to be in 5.0.3 to allow time for translations to get to the spreadsheet and get translated.